security/backends/elasticsearch/writer.py

```diff
@@ -37,6 +38,8 @@
 
 
 logstash_logger = logging.getLogger('security.logstash')
+vector_logger = structlog.getLogger("security.vector")
```

Comment:
Using structlog is not needed for this change. 

We may also serialize data into JSON manually - the same way we do it for Logstash.  I used structlog because it is our long-term initiative to replace all handlers manually constructing JSON with structlog.

As I was reviewing these changes though I came to the conclusion that this is perhaps premature optimization.

Reply:
I am kind of conflicted here. IMO logging `json.dumps` manually seems like a hack.

Reply:
Yeah, in first phase we could probably use standard logging, but I guess it wouldn't save much time during implementation.